Typescript Warning / Error

```
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
    at CommonJsRequireContextDependency.getWarnings (/tunnel/node_modules/webpack/lib/dependencies/ContextDependency.js:40:18)
    at Compilation.reportDependencyErrorsAndWarnings (/tunnel/node_modules/webpack/lib/Compilation.js:1454:24)
    at /tunnel/node_modules/webpack/lib/Compilation.js:1258:10
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/tunnel/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:22:1)
    at AsyncSeriesHook.lazyCompileHook (/tunnel/node_modules/tapable/lib/Hook.js:154:20)
    at Compilation.finish (/tunnel/node_modules/webpack/lib/Compilation.js:1253:28)
    at /tunnel/node_modules/webpack/lib/Compiler.js:672:17
    at eval (eval at create (/tunnel/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /tunnel/node_modules/webpack/lib/Compilation.js:1185:12
    at /tunnel/node_modules/webpack/lib/Compilation.js:1097:9
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
 @ ./node_modules/ts-results/index.js
 @ ./node_modules/@elbstack/lexoffice-client-js/dist/index.js
 @ ./src/resolvers/quotation.ts
 @ ./src/voucher-sync.ts
 @ ./src/index.ts, ./node_modules/ts-results/option.js 3:24-31
```

Typescript version: 4.3.4
Ts-result version: 3.3.0

I want to deploy a atlassian forge app, and on the snapshot function this error will be thrown:

```
Error: Error thrown in the snapshot context.
App code snapshot error: Cannot find module 'tslib'
Learn more about the snapshot context at http://go.atlassian.com/forge-snapshot-context.
```

Looks like the same error as #19

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -15,7 +15,6 @@
     "baseUrl": "./",
     // NOTE: Don't change this from es5 - we use this to allow callable classes.
     "target": "es5",
-    "importHelpers": true,
     "forceConsistentCasingInFileNames": true,
     "experimentalDecorators": true
   },

